roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/TaskToolbarPanel.kt

```diff
@@ -52,31 +43,20 @@ class StatusToolbarPanel(
 
   init {
     actionToolbar.setTargetComponent(this)
-    setBorder(BorderFactory.createEmptyBorder(5, 5, 5, 5))
-    val gbc = GridBagConstraints().apply {
-        gridx = 0
-        gridy = 0
-        anchor = GridBagConstraints.WEST
-        insets = JBUI.insets(5)
+    setBorder(BorderFactory.createEmptyBorder(4, 4, 4, 4))
+    val actionToolbarConstraints = GridBagConstraints().apply {
+      insets = JBUI.insets(4)
+      gridx = 0
+      gridy = 0
+      anchor = GridBagConstraints.WEST
     }
-    add(_statusLabel, gbc)
-    val hRGlueConstraints = gbc.apply {
+    add(actionToolbar.component, actionToolbarConstraints)
+    add(JBBox.createHorizontalGlue(), GridBagConstraints().apply {
```

Comment:
I believe now that we have the status label moved to the bottom we could just use a `BorderLayout` so we don't have to set all these constraints and the horizontal glue.

Minor: Also I foresee the border still showing when the `actionToolbar` visibility is toggled to `false`. We could toggle the JBox instead that's if we wrap the `actionToolbar` into the JBBox's horizontal box. Something along the lines:

```kotlin
...
 private val jbBox = JBBox.createHorizontalBox()

  init {
    actionToolbar.setTargetComponent(this)
    add(jbBox.apply {
      isVisible = false
      setBorder(BorderFactory.createEmptyBorder(4, 4, 4, 4))
      add(actionToolbar.component, GridBagConstraints().apply {
        gridx = 0
        gridy = 0
        anchor = GridBagConstraints.WEST
        insets = JBUI.insets(4)
      })
    }, BorderLayout.WEST)
  }

  fun setActions(actions: List<ToolbarAction>) {
    if (actions.isEmpty()) {
      jbBox.isVisible = false
      return
    }
    jbBox.isVisible = true
    val actionList = listOf(*actions.toTypedArray())
    actionToolbar.setActions(actionList)
  }
...  
```

Reply:
Thanks for your advice. I released it a few days ago. I want to include these changes, but I'm a little busy. I'll do this the next time I modify the plugin, but if you could do it, it would be highly appreciated.

Reply:
Sure. I'll look into it.